helm_deploy/whereabouts-api/values.yaml

```diff
@@ -45,7 +45,6 @@ generic-service:
       NOTIFY_API_KEY: "NOTIFY_API_KEY"
     feature-toggles:
       FEATURE_LISTEN_FOR_COURT_EVENTS: "FEATURE_LISTEN_FOR_COURT_EVENTS"
-      FEATURE_EVENT_WHEREABOUTS_API_VIDEOLINK_MIGRATE: "FEATURE_SEND_VIDEO_BOOKING_MIGRATE_EVENTS"
```

Comment:
This can be removed from the dev, preprod and, prod namespaces once deployed to each env.